statescript/executor.go

```diff
@@ -28,6 +28,17 @@ import (
 	"github.com/pkg/errors"
 )
 
+const (
+	exitRetryLater = 21
+
+	defaultRetryWindow time.Duration = 5 * time.Second // Total time given to a retry-script
```

Comment:
Shouldn't retry window be longer than retry timeout? As for the defaults, I think we can do something like: retry every 60 seconds and timeout 30 minutes. What I am also thinking is (we can open separate JIRA ticket) we can have `0` timeout value which will let the script wait forever (might be risky though).

Reply:
https://tracker.mender.io/browse/MEN-1479